view/keyboard_widget.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/wtfutil/wtf/utils"
 )
 
-const helpKeyChar = "/"
+const helpKeyChar = "?"
```

Comment:
I like everything about this, except this change. We can't change fundamentals like this on people without a really good reason. It'll break people's expectations and muscle memory.

Without this, I'm happy to merge this PR in.

Reply:
You're right, sorry about that. I've been sick the past couple days, but I'll fix this up once I'm better + one small issue I noticed also needs to be done. Should be hopefully tomorrow or day after

Reply:
Sounds good. Feel better!